code/src/java/plugin/network/NetworkClient.java

```diff
@@ -19,40 +19,40 @@
  */
  package plugin.network;
 
-import pcgen.core.SettingsHandler;
-import pcgen.util.Logging;
-
 import java.io.BufferedOutputStream;
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
 import java.io.PrintStream;
 import java.net.Socket;
 import java.util.StringTokenizer;
 
-public class NetworkClient
+import pcgen.core.SettingsHandler;
+import pcgen.util.Logging;
+
+class NetworkClient
```

Comment:
This needs testing.  I think this breaks the plugins so they can no longer be loaded.
I've already had to revert a similar change elsewhere in the plugin system.